data/data/openstack/masters/sg-master.tf

```diff
@@ -264,13 +281,14 @@ resource "openstack_networking_secgroup_rule_v2" "master_ingress_https" {
 }
 
 resource "openstack_networking_secgroup_rule_v2" "master_ingress_router" {
+  count             = length(var.machine_v4_cidrs)
   count             = var.masters_schedulable ? 1 : 0
```

Comment:
Is this count needed?
It's conflicting with the new one defined as they have the same name.

Reply:
Good catch. I'll fix it.